src/utils/dom/createNode.js

```diff
@@ -23,12 +24,18 @@ export default function createNode(
   Object.keys(attrs).forEach(key => {
     const value = attrs[key];
 
-    if (typeof value === "function") {
+    if (key === "text") {
+      result.innerText = value;
```

Comment:
I feel like this is a bit misleading, since `text` comes through as an attribute but sets the `innerText` property. I would rather see `createNode` accept both `attributes` and `properties` options and then the consuming module would pass `{ innerText: "some value" }` for the `properties` option. I would expect any values that are functions to likewise be passed on the `properties` option instead of having the `isFunction` condition below.

Reply:
I have added `props` to `createNode()` and removed the `isFunction` altogether. The `fireImage` module has been adjusted to pass in `onload`, `onerror` and `onabort` as `props` instead of `attrs`.